py3status/modules/deadbeef.py

```diff
@@ -3,95 +3,119 @@
 Display song currently playing in deadbeef.
 
 Configuration parameters:
-    cache_timeout: how often we refresh usage in seconds (default 1)
-    delimiter: delimiter character for parsing (default '¥')
-    format: see placeholders below (default '{artist} - {title}')
+    cache_timeout: refresh interval for this module (default 1)
+    delimiter: delimiter between metadata fields (default 'ÐÉÅÐƁÊËF')
+    format: display format for this module (default '[{artist} - ][{title}]')
 
 Format placeholders:
-    {artist} artist
-    {title} title
-    {elapsed} elapsed time
-    {length} total length
-    {year} year
-    {tracknum} track number
+    {artist} Name of the artist of the track.
+    {isplaying} "1" if file is currently playing, empty string otherwise.
```

Comment:
We missed this placeholder... I added it just now.. for that guy who wanted this in.